### ISSUE TITLE
[需求] Is better for  FastBeanCopier  change CACHE type  to ConcurrentReferenceHashMap

### ISSUE THREAD

**Issue by clivetao** [hs-web/hsweb-framework#186]
f in a scenario where the key changes frequently,ConcurrentHashMap  takes up a lot of invalid memory

**Comment by zhou-hao** [comment]
I don't think so ,Creating Copier is time-consuming, and using ConcurrentHashMap may speed up program crashes!

**Comment by clivetao** [comment]
> I don't think so ,Creating Copier is time-consuming, and using ConcurrentHashMap may speed up program crashes!

emmm, maybe you say ConcurrentReferenceHashMap will speed up program crashes ?
 FastBeanCopier exactly use ConcurrentHashMap here !
You have a point, It's cost-effective to have high performance requirements ! But the situations of out-of-memory situations also need to be considered , is there can provider a choose .
![截屏2021-12-01 下午4 51 49](https://user-images.githubusercontent.com/38341571/144202246-4e56523c-d73b-47a9-8ea4-1ab07871972d.png)

**Comment by zhou-hao** [comment]
You are right , It should be configurable , can you pull request to do it ?

**Comment by clivetao** [comment]
> 

Thanks for your invitation , i will try do this !
